src/all/comicklive/src/eu/kanade/tachiyomi/extension/all/comicklive/Comick.kt

```diff
@@ -251,32 +279,50 @@ class Comick(
         } catch (e: Throwable) {
             Log.e(name, "Unable to parse filters", e)
 
+            if (!getTags) {
+                filters.addAll(
+                    index = 2,
+                    textTags,
+                )
+            }
             filters.addAll(
                 index = 0,
                 listOf(
-                    Filter.Header("Failed to parse genres and tags"),
+                    Filter.Header("Failed to parse genres ${if (getTags) "and tags" else ""}"),
                     Filter.Separator(),
                 ),
             )
             return@runBlocking FilterList(filters)
         }
 
-        filters.addAll(
-            index = 1,
-            listOf(
-                GenreFilter(data.genres),
-                TagFilter(data.tags),
-            ),
+        filters.add(
+            index = 3,
+            GenreFilter(data.genres),
         )
+        if (!getTags) {
+            filters.addAll(
+                index = 4,
+                textTags,
+            )
+        } else {
+            filters.add(
+                index = 4,
+                TagFilter(data.genres),
+            )
+        }
         return@runBlocking FilterList(filters)
     }
 
     override fun mangaDetailsRequest(manga: SManga) =
         GET("$baseUrl/comic/${manga.url}", headers)
 
+    private val altTitleRegex = Regex(""""md_titles":\{((?:"\d+":\{.*?\},?)+)\}""")
```

Comment:
if they serve either a map or list, then use a `JsonTransformingSerializer` in the dto to convert the map to list on deserialization